[EPPETA-62]  Validation on EvaluationForm

jleiva-gap opened the pull request:
Highlight and show a message for select values required

jleiva-gap committed:
[EPPETA-62]  Validation on EvaluationForm

Highlight and show a message for select values required

jleiva-gap committed:
Fix style problem

stevenarnold-ed-fi commented on src/reactapp/src/pages/EvaluationForm.js:
```diff
@@ -346,11 +346,39 @@ export default function EvaluationForm() {
+   * @returns
+   */
+  const areAllScoreSelected = () => {
+    let hasPendingScores = false;
```
@jleiva-gap 'Not Assessed' is a valid selection for the evaluator to choose, but if the evaluator wants to choose 'Not Assessed' they need to choose it, not have it defaulted. I think the best solution here is to have a blank choice and that becomes the -1/invalid choice.

stevenarnold-ed-fi reviewed (changes requested):
@jleiva-gap This

jleiva-gap committed:
Update EvaluationForm.js

Update the form to load by default not Assessed when is loading an existing evaluation
